common/Perf/Azure.Test.Perf/PerfProgram.cs

```diff
@@ -174,6 +178,48 @@ private static async Task Run(Type testType, PerfOptions options)
             PrintAssemblyVersions(testType);
         }
 
+        private static void ConfigureThreadPool(PerfOptions options)
+        {
+            if (options.MinWorkerThreads.HasValue || options.MinCompletionPortThreads.HasValue)
+            {
+                ThreadPool.GetMinThreads(out var minWorkerThreads, out var minCompletionPortThreads);
+                ThreadPool.SetMinThreads(options.MinWorkerThreads ?? minWorkerThreads,
```

Comment:
Since the setter calls can return `false`, indicating the value was not updated, should we inform the user clearly that their desired config was not applied.
It will be printed via `PrintEnvironment()` but it seems like this should should show up as an error.

Reply:
I never knew these APIs had a return value.  I agree the perf framework should fail fast if one of these happens to return `false`.